### DISCUSSION TITLE
Log Warnings: Unused ODF entries

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#1369] [Help]
While loading the Bureå church sample sets, I get a lot of Warnings of Unused ODF entries. All stops are functional.

Where did I go wrong? 

(This questions is perhaps most directed towards @larspalo)

**Comment by larspalo** [comment]
@jason-barba It's likely that you did nothing wrong at all. Bureå church sample set is my first real attempt at creating a sample set (with the help of Graham Goode). Thus, a lot has happened since then that makes it more obvious that there are problems in that odf. One way to reduce the reported errors is to disable Strict checking of odf. However, I hope to create an updated package of that sample set. Also, in the future I'd like to create a new version of the organ in Bureå church that reflect the current state of the organ that has slightly changed from when the first samples were recorded. In the mean time make sure you have applied all the patches available.

**Reply by jason-barba** [reply]
OK, so I turn a blind eye ... ;-)
Is the patch applying order significant?

**Reply by larspalo** [reply]
No, I don't think it should be, but each patch will either overwrite some files or provide new ones.

**Reply by jason-barba** [reply]
Thanks for advice. Patches are installed and warning messages have disappeared ;-)
